### PR TITLE
Image metadata dl

### PR DESCRIPTION
**Pull request by tsop14**
#993

Changes the `<table>` values to `<dl>` for the metadata on selection view and image view, and the drop down full-metadata list on image view.

### CODE REVIEW

File: kahuna/public/stylesheets/main.css [hunk 1]
```diff
@@ -1099,6 +1099,57 @@ FIXME: what to do with touch devices
     position: relative;
 }
 
+.image-info__group--dl {
+  width: 100%;
+  display: -webkit-box;
+  display: -webkit-flex;
```
Comment (theefer) [inline kahuna/public/stylesheets/main.css]: Is that really necessary for the browsers we support (IIRC Chrome 39+, FF 33+)?

File: kahuna/public/js/components/gr-panel/gr-panel.html [hunk 2]
```diff
@@ -175,78 +175,72 @@ <h2 class="image-info__title" ng:if="!ctrl.hasMultipleValues(ctrl.rawMetadata.ti
             </div>
         </div>
```
Comment (theefer) [inline kahuna/public/js/components/gr-panel/gr-panel.html]: Do we also want to use DL for title/description/special instructions?

**Comment by theefer** [comment]
:+1:

**Comment by jamesgorrie** [comment]
Side note - [some nice gitness](https://github.com/blog/1506-closing-issues-via-pull-requests)

**Pull request merged**